src/App.jsx

```diff
@@ -1,7 +1,36 @@
 import ReactDOM from "react-dom/client";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import AddPage from "./pages/AddPage/AddPage";
+import AllPage from "./pages/AllPage/AllPage";
+import CompletedPage from "./pages/CompletedPage/CompletedPage";
+import UnCompletedPage from "./pages/UnCompletedPage/UnCompletedPage";
+import ConfigurationPage from "./pages/ConfigurationPage/ConfigurationPage";
+import Main from "./components/Navbar/Navbar";
+import Header from "./components/Header/Header";
+import Navbar from "./components/Navbar/Navbar";
 
 const App = () => {
-
+  return (
+    <BrowserRouter>
+      <>
+        <div className="site-wrapper">
+          <div className="container">
+            <main className="content">
+              <Header />
+              <Navbar />
+              <Routes>
+                <Route path="/add" element={<AddPage />} />
+                <Route path="/" element={<AllPage />} />
+                <Route path="/completed" element={<CompletedPage />} />
+                <Route path="/uncompleted" element={<UnCompletedPage />} />
```

Comment:
These 3 routes should use the same component. The reason is that, they responsibility is to render the list of todos. The only thing that is different is the data inside the component should be filtered.

Reply:
Done.